patsylearn/tests/test_patsy_model.py

```diff
@@ -106,6 +107,20 @@ def test_stateful_transform():
     # make sure that mean of training, not test data was removed
     assert_array_equal(data_trans[:, 0], -1)
 
+def test_stateful_transform_dataframe():
```

Comment:
anyhow, I guess the test is sufficient, I just find it a bit confusing why you test what you tests.